sdk/communication/communication-administration/test/utils/recordedClient.ts

```diff
@@ -38,16 +38,16 @@ export const environmentSetup: RecorderEnvironmentSetup = {
   customizationsOnRecordings: [
     (recording: string): string =>
       recording.replace(/"token"\s?:\s?"[^"]*"/g, `"token":"sanitized"`),
-    (recording: string): string => recording.replace(/(https:\/\/)([^\/',]*)/, "$1endpoint"),
+    (recording: string): string => recording.replace(/(https:\/\/)([^/',]*)/, "$1endpoint"),
     (recording: string): string => recording.replace(/"id"\s?:\s?"[^"]*"/g, `"id":"sanitized"`),
     (recording: string): string => {
       return recording.replace(
-        /(https:\/\/[^\/',]*\/identities\/)[^\/',]*(\/token)/,
+        /(https:\/\/[^/',]*\/identities\/)[^/',]*(\/token)/,
```

Comment:
This doesn't look right, regex101.com at least complains that this the pattern is invalid
![image](https://user-images.githubusercontent.com/1885005/108430671-93c53300-71f6-11eb-92e1-800a99da99e4.png)